react-native/react/store/configure-store.native.desktop.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+/* @flow */
+
+import { createStore, compose } from 'redux'
+import { devTools } from 'redux-devtools'
+
+export default function configureStoreNative (rootReducer, initialState, createStoreWithMiddleware) {
+  const nativeCreateStoreWithMiddleware = compose(
+    createStoreWithMiddleware,
+    devTools()
+  )(createStore)
+
+  return nativeCreateStoreWithMiddleware(rootReducer, initialState)
+}
```

Comment:
I think this creates more implied stuff coming from the parent, but its fine for now